src/labeler.ts

```diff
@@ -22,75 +22,80 @@ export async function run() {
     const syncLabels = !!core.getInput('sync-labels');
     const dot = core.getBooleanInput('dot');
 
-    const prNumber = getPrNumber();
-    if (!prNumber) {
-      core.info('Could not get pull request number from context, exiting');
+    const prNumbers = getPrNumbers();
+    if (!prNumbers) {
+      core.info('Could not get pull request number, exiting');
       return;
     }
 
     const client: ClientType = github.getOctokit(token, {}, pluginRetry.retry);
 
-    const {data: pullRequest} = await client.rest.pulls.get({
-      owner: github.context.repo.owner,
-      repo: github.context.repo.repo,
-      pull_number: prNumber
-    });
-
-    core.debug(`fetching changed files for pr #${prNumber}`);
-    const changedFiles: string[] = await getChangedFiles(client, prNumber);
-    const labelGlobs: Map<string, StringOrMatchConfig[]> = await getLabelGlobs(
-      client,
-      configPath
-    );
-
-    const preexistingLabels = pullRequest.labels.map(l => l.name);
-    const allLabels: Set<string> = new Set<string>(preexistingLabels);
-
-    for (const [label, globs] of labelGlobs.entries()) {
-      core.debug(`processing ${label}`);
-      if (checkGlobs(changedFiles, globs, dot)) {
-        allLabels.add(label);
-      } else if (syncLabels) {
-        allLabels.delete(label);
-      }
-    }
-
-    const labelsToAdd = [...allLabels].slice(0, GITHUB_MAX_LABELS);
-    const excessLabels = [...allLabels].slice(GITHUB_MAX_LABELS);
-
-    try {
-      let newLabels: string[] = [];
+    for (const prNumber of prNumbers) {
+      const {data: pullRequest} = await client.rest.pulls.get({
+        owner: github.context.repo.owner,
+        repo: github.context.repo.repo,
+        pull_number: prNumber
+      });
 
-      if (!isListEqual(labelsToAdd, preexistingLabels)) {
-        await setLabels(client, prNumber, labelsToAdd);
-        newLabels = labelsToAdd.filter(l => !preexistingLabels.includes(l));
+      if (!pullRequest) {
+        core.warning(`Could not find pull request #${prNumber}, skipping`);
+        continue;
```

Comment:
If the PR does not exist, we will get an error here. We should add the `try` block here to handle the error and move the content of the `if` statement to the `catch` block.